Composer/packages/ui-plugins/virtual-assistant-creation/src/components/customizeBotPage.tsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable format-message/literal-pattern */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { useContext } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { Dropdown, IDropdownStyles, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import { Label } from 'office-ui-fabric-react/lib/Label';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+
+import { RouterPaths } from '../shared/constants';
+import { AvailablePersonalities } from '../models/creationOptions';
+
+import { AppContext } from './VirtualAssistantCreationModal';
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+
+// -------------------- Styles -------------------- //
+const textFieldStyling = css`
+  width: 400px !important;
+`;
+
+const horizontalStackTokens = { childrenGap: 10 };
+const verticalStackTokens = { childrenGap: 15 };
+
+const dropdownStyles: Partial<IDropdownStyles> = {
+  dropdown: { width: 300 },
+};
+
+// -------------------- CustomizeBotPage -------------------- //
+type CustomizeBotPageProps = {
+  onDismiss: () => void;
+} & RouteComponentProps<{}>;
+
+export const CustomizeBotPage: React.FC<CustomizeBotPageProps> = (props) => {
+  const { state, setState } = useContext(AppContext);
+  const onDismiss = props.onDismiss;
+
+  const personalityOptions = (): IDropdownOption[] => {
+    return Object.values(AvailablePersonalities).map((personality) => {
+      return { key: personality, text: formatMessage(personality) };
```

Comment:
I'm uncertain if this will end up putting each personality into the en-us.json file.
@beyackle - does formatMessage handle runtime calls in a loop to formatMessage?

Reply:
It does not. `formatMessage` must be called with a string literal.

Reply:
Changed the syntax to support variable string